addons/web_dashboard/models/res_users.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+
+from datetime import datetime, timedelta
+
+from openerp import models, api, _
+from openerp.exceptions import RedirectWarning
+from openerp.tools.misc import DEFAULT_SERVER_DATETIME_FORMAT
+
+
+class Users(models.Model):
+    _inherit = 'res.users'
+
+    @api.model
+    def web_dashboard_create_users(self, emails, optional_message=""):
+        existing_users = self.search(['&', ('login', 'in', emails), '|', ('active', '=', True), ('active', '=', False)])
+
+        # Processing existing users
+        for user in existing_users:
+            if not user.login_date and not user.signup_valid:
+                user.action_reset_password()
+            else:
+                # Already a user, reactivated in case it is deactivated
+                user.active = True
+        new_emails = set(emails) - set(existing_users.mapped('login'))
+        template_user_id = self.env['ir.config_parameter'].get_param('auth_signup.template_user_id', 'False')
+
+        # Processing new email addresses : create new users (Used template user)
+        for email in new_emails:
+            default_values = {'login': email, 'name': email, 'email': email, 'active': True}
+            if template_user_id:
+                user = self.with_context(custom_message=optional_message, create_user=1).browse(int(template_user_id)).copy(default=default_values)
+            else:
+                user = self.with_context(custom_message=optional_message, create_user=1).create(default_values)
+            # If user haven't configured Outgoing mail server properly
+            if not user.signup_token:
```

Comment:
Ok, thanks. The redirect message with configuration of mail server is a good thing, I just wanted to be sure that it cannot be related to another problem.